### PR TITLE
Flake8 package checking python coding style: test_mkldnn.py, small tidy up

### PR DESCRIPTION
**Pull request by mozga-intel**
## Description ##
This pull request tidies up the python unit-test file.  

## Confuguration ##
Flake8 package is required to get a result like please have a look: that: https://flake8.pycqa.org/en/latest/#. The main dir contains .flake8 file with a configuration, as follows: [Example]
```cpp
[flake8]
select = B,C,E,F,P,T4,W,B9
max-line-length = 120
ignore =
    E203,E305,E402,E501,E721,E741,F405,F821,F841,F999,W503,W504,C408,E302,W291,E303,F403
    EXE001,
    B007,B008, E722
    C400,C401,C402,C403,C404,C405,C407,C411,C413,C414,C415
optional-ascii-coding = True
exclude = 
    ./.git,
    ./.build,
    *.pyi
```

## Future consideration ##
This process should be added to a hook and it could be run before any commit (run lint). Additionally, CI should have a mechanism to detect differences between versions.  

### Essentials ###
- [ ] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

File: tests/python/mkl/test_mkldnn.py [hunk 1]
```diff
@@ -309,15 +319,18 @@ def check_batchnorm_training(stype):
             data = mx.symbol.Variable('data', stype=stype)
             in_location = [mx.nd.array(data_tmp).tostype(stype), mx.nd.array(gamma).tostype(stype),
                            mx.nd.array(beta).tostype(stype)]
-            mean_std = [mx.nd.array(rolling_mean).tostype(stype), mx.nd.array(rolling_std).tostype(stype)]
+            mean_std = [mx.nd.array(rolling_mean).tostype(
```
Comment (anko-intel) [inline tests/python/mkl/test_mkldnn.py:322]:
```suggestion
                        mean_std = [mx.nd.array(rolling_mean).tostype(stype), 
                                    mx.nd.array(rolling_std).tostype(stype)]
```
or 
```suggestion
                        mean_std = 
                                   [mx.nd.array(rolling_mean).tostype(stype), 
                                    mx.nd.array(rolling_std).tostype(stype)]
```
?

File: tests/python/mkl/test_mkldnn.py [hunk 2]
```diff
@@ -405,14 +420,18 @@ def check_pooling_training(stype):
             in_location = [mx.nd.array(data_tmp).tostype(stype)]
 
             if np.array(shape).shape[0] == 3:
-                test = mx.symbol.Pooling(data=data, kernel=(3), stride=(2), pool_type='avg')
+                test = mx.symbol.Pooling(data=data, kernel=(
+                    3), stride=(2), pool_type='avg')
             elif np.array(shape).shape[0] == 4:
-                test = mx.symbol.Pooling(data=data, kernel=(3, 3), stride=(2, 2), pool_type='avg')
+                test = mx.symbol.Pooling(data=data, kernel=(
+                    3, 3), stride=(2, 2), pool_type='avg')
```
Comment (anko-intel) [inline tests/python/mkl/test_mkldnn.py:427]: It looks strange when part of the parameter is moved to the next line. "(3, 3)" shouldn't' be spitted

**Review by anko-intel — CHANGES_REQUESTED** [review]

File: tests/python/mkl/test_mkldnn.py (hunk 1 above)
Comment (mozga-intel) [reply tests/python/mkl/test_mkldnn.py]: Please have a look at this ~ there are a few fixes

File: tests/python/mkl/test_mkldnn.py (hunk 2 above)
Comment (mozga-intel) [reply tests/python/mkl/test_mkldnn.py]: Indeed, max limit was added max-line-length = 120. Should be fine right now. Please have a look at that.

**Review by sfraczek — APPROVED** [review]
LGTM

**Review by anko-intel — APPROVED** [review]

**Comment by mozga-intel** [comment]
@mxnet-bot run ci [unix-gpu]

**Comment by mozga-intel** [comment]
@mxnet-bot run ci [unix-gpu]

**Comment by mozga-intel** [comment]
@mxnet-bot run ci [unix-gpu]

**Review by akarbown — APPROVED** [review]

**Pull request merged**